packages/@webex/internal-plugin-metrics/src/call-diagnostic/call-diagnostic-metrics.ts

```diff
@@ -226,11 +227,17 @@ export default class CallDiagnosticMetrics extends StatelessWebexPlugin {
       const meeting = this.webex.meetings.meetingCollection.get(meetingId);
 
       if (!meeting) {
-        // TODO: add behavioral metrics to see if this actually happens in production.
         console.warn(
           'Attempt to send MQE but no meeting was found...',
           `event: ${name}, meetingId: ${meetingId}`
         );
+        // @ts-ignore
+        this.webex.internal.metrics.submitClientMetrics(CALL_DIAGNOSTIC_EVENT_FAILED_TO_SEND, {
```

Comment:
do you think the meeting id is sufficient here , they are just correlation Ids ?

Reply:
I could add correlation ID too, but not sure how to grab it? (since I don't have access to the meeting object)